site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -120,6 +120,7 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `agones.controller.healthCheck.timeoutSeconds`      | Number of seconds after which the probe times out (in seconds)                                  | `1`                    |
 | `agones.controller.resources`                       | Controller [resource requests/limit][resources]                                                 | `{}`                   |
 | `agones.controller.generateTLS`                     | Set to true to generate TLS certificates or false to provide your own certificates              | `true`                 |
+| `agones.controller.reuseGeneratedTLS`               | Set to true to reuse generated TLS certificates with lookup existing secrets                    | `true`                 |
```

Comment:
For newly added config params, please add them to the table **New Configuration Features** (down around line 211). This way they won't be published to the website until the next release, when the helm chart will be published that supports using them.

Reply:
This also seems like a change in behaviour - i.e. we currently regen certs, and this means that it won't.

I would advocate for a default of `false`, so that there isn't a breaking change in behaviour in the helm charts.

Reply:
Okay, working on it

Reply:
done